### PR TITLE
fix: Navbar section updated for mobile view

### PR DESCRIPTION
**Pull request by AbhiDiva96**
## Related Issue

fixes: #305 

## Description

  Improved/enhanced the navbar section for the mobile view

## Before change

Screenshot 2024-05-17 161932

## After change
Screenshot 2024-05-17 162619

## Added theme button so that user can easily shift from dark to light mode or vice-versa
Screenshot 2024-05-17 160412

## Summary by CodeRabbit

- **Improvements**
  - Enhanced the layout and styling of the Navbar for a better visual presentation and user experience on both desktop and mobile views.
  - Minor formatting adjustments to the Theme Button for code consistency.

### CODE REVIEW

**Comment by Sanchitbajaj02** [comment]
@AbhiDiva96 why this is raised and not included into your previous branch?

**Comment by AbhiDiva96** [comment]
I don't want to mess up all the things on that PR. I will close that PR. When you review this one.
@Sanchitbajaj02

**Comment by AbhiDiva96** [comment]
Hey @Sanchitbajaj02 
Pls review this PR

**Comment by Sanchitbajaj02** [comment]
> Hey @Sanchitbajaj02 Pls review this PR

PR is pulled locally and in review process. It will be merged today

**Comment by AbhiDiva96** [comment]
> > Hey @Sanchitbajaj02 Pls review this PR
> 
> PR is pulled locally and in review process. It will be merged today



> Hey @Sanchitbajaj02 Pls review this PR



> > Hey @Sanchitbajaj02 Pls review this PR
> 
> PR is pulled locally and in review process. It will be merged today

thankyou so much @Sanchitbajaj02

**Comment by AbhiDiva96** [comment]
thank-you so much
@Sanchitbajaj02

File: package.json [hunk 1]
```diff
@@ -63,7 +63,8 @@
     "sharp": "0.33.1",
     "sonner": "^1.4.0",
     "tailwindcss": "^3.4.1",
-    "typescript": "5.1.6"
+    "typescript": "5.1.6",
+    "yarn": "^1.22.22"
```
Comment (Sanchitbajaj02) [inline package.json:67]: remove this from dependencies

File: src/components/core/navbar/index.tsx [hunk 2]
```diff
@@ -129,7 +129,14 @@ const Navbar = ({ starCount }: { starCount?: number }) => {
           </Link>
 
           {/* Hamburger menu button for small screens */}
-          <div className="md:hidden flex items-center">
+
+         
+
+          <div className="md:hidden flex justify-end">
+          <div className="flex justify-end">
```
Comment (Sanchitbajaj02) [inline src/components/core/navbar/index.tsx:136]: we don't need extra flexboxes

**Review by Sanchitbajaj02 — CHANGES_REQUESTED** [review]

**Comment by AbhiDiva96** [comment]
@Sanchitbajaj02 
please consider this PR as the final one. on forward I am not going to make any other PR. I have pushed all the changes in this one. so sorry for bothering you.

**Comment by Sanchitbajaj02** [comment]
> @Sanchitbajaj02 please consider this PR as the final one. on forward I am not going to make any other PR. I have pushed all the changes in this one. so sorry for bothering you.

I added reviews in this pr as well as in your old PRs. Please do that to get it merged. If you're facing problems related to raising prs and git please reach out to me on discord

**Comment by Sanchitbajaj02** [comment]
@AbhiDiva96 Please acknowledge the changes that I have raised. If you have any doubt, please respond. I will close this PR in 48hrs if you didn't

**Comment by AbhiDiva96** [comment]
This is the final PR please merge it 
@Sanchitbajaj02

**Comment by Sanchitbajaj02** [comment]
> I closed the initial two PR. This is the final PR please review and merge it this PR waiting for reviews pls merge it. @Sanchitbajaj02

Hi @AbhiDiva96 i have told many times above that you have to change some things to get it merged. Please fix those things then I will merge. It is not just that you create something and I merge it blindly

**Comment by AbhiDiva96** [comment]
dear @Sanchitbajaj02 
i have added all your requirements. which you gave to me in the above. i have add a commit name "improve ui of navbar in mobile view". if any query ping me 

image

**Comment by Sanchitbajaj02** [comment]
> dear @Sanchitbajaj02 i have added all your requirements. which you gave to me in the above. i have add a commit name "improve ui of navbar in mobile view". if any query ping me
> 
> ## image of commit
> image

take latest update from master since it has some merge conflicts. Fix them and I will merge

**Pull request merged**